resources/views/website/layouts/assets/javascripts.blade.php

```diff
@@ -352,6 +352,33 @@ function updateCountdownGetTimestring(delta) {
 
 </script>
 
+<script type="text/javascript">
+    $(function() {
+        if (sameDay(new Date(), new Date('April, 1, 2021'))) {
+            for (let img of document.getElementsByTagName('img')) {
+                if (img.src.includes('profile_images') || img.src.includes('default-avatar')) {
+                    img.src = 'https://pbs.twimg.com/profile_images/1092451626781163523/0YzJMi-8.jpg';
+                } else if (img.src.includes('images/logo/inverse.png')) {
+                    img.src = '/images/logo/broto-inverse.png'
+                } else if (img.src.includes('images/logo/regular.png')) {
+                    img.src = '/images/logo/broto-regular.png'
+                }
+            }
+
+            $("body").children().each(function () {
+                console.log($(this).html());
+                $(this).html($(this).html().replace('Proto', ' Broto').replace('Proto.', ' Broto.'));
```

Comment:
Waarom twee keer this.html?

Reply:
Dit heb jij geschreven. Maar het is gewoon de html replacen met dezelfde content met Proto replaced.